larch.js

```diff
@@ -41,8 +41,12 @@ function Larch(options) {
 
     if (self.backends.length === 1) {
         self.log = self.logSingleBackend;
```

Comment:
What's the most optimum v8 way of doing this ?

This branch creates two different hidden classes.

Is this static branching actually faster then an `if` statement inside the method ?